assets/js/components/rank-labels.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import _ from "lodash";
+
+class RankLabels extends React.Component {
+  get rankLabels() {
+    return [1, 2, 3, 4, 5, 6, 7, 8];
+  }
+
+  renderRankLabels() {
+    return _.map(this.rankLabels, (rankLabel) => {
```

Comment:
Doesn’t JavaScript have a native map we can utilize these days?

Reply:
It does, but it doesn't work in PhantomJS.